aardvark/updater/__init__.py

```diff
@@ -1,34 +1,41 @@
-#ensure absolute import for python3
-from __future__ import absolute_import
-
 import copy
+import logging
 import time
 
 from blinker import Signal
+from cloudaux.aws.decorators import rate_limited
 from cloudaux.aws.iam import list_roles, list_users
 from cloudaux.aws.sts import boto3_cached_conn
-from cloudaux.aws.decorators import rate_limited
+
+from aardvark.plugins import AardvarkPlugin
+
+log = logging.getLogger("aardvark")
 
 
-class AccountToUpdate(object):
+class AccountToUpdate(AardvarkPlugin):
     on_ready = Signal()
```

Comment:
Are these signals used anywhere?

Reply:
Not in this codebase. They're meant for integration/customization, so we have some handlers in our internal codebase.